core/src/main/java/com/sequenceiq/cloudbreak/service/ldapconfig/LdapConfigService.java

```diff
@@ -105,15 +105,20 @@ private void delete(LdapConfig ldapConfig) {
         authorizationService.hasWritePermission(ldapConfig);
         List<Cluster> clustersWithLdap = clusterRepository.findByLdapConfig(ldapConfig);
         if (!clustersWithLdap.isEmpty()) {
-            StringBuilder ldaps = new StringBuilder();
-            ldaps.append('[');
-            for (int i = 0; i < clustersWithLdap.size() - 1; i++) {
-                ldaps.append(clustersWithLdap.get(i).getName()).append(", ");
+            if (clustersWithLdap.size() > 1) {
```

Comment:
!isEmpty() ?

Reply:
the 108th line checks if the list contains *more* than one element, to separate the different type of messages (different one comes back if only one cluster related, and another one if more than one)